runtime/onert/backend/gpu_cl/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(LIB_ONERT_BACKEND_GPU_CL onert_backend_gpu_cl)
+
+nnfw_find_package(ARMCompute QUIET)
+if(NOT ARMCompute_FOUND)
+  return()
+endif(NOT ARMCompute_FOUND)
+
+# find_package(abseil REQUIRED)
```

Comment:
If it is not necessary comment, Please remove..

Reply:
okay I will remove.